scripts/kubernetes-all-test.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up some stuff"
+
+# Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
+mkdir bin artifacts
+# Change the default location of go's bin directory (without affecting GOPATH). This is where compiled binaries will end up by default
+# for eg go get ginkgo later on will produce ginkgo binary in GOBIN
+export GOBIN="`pwd`/bin"
+
+# Set kubeconfig to current dir. This ensures no clashes with other test runs
+export KUBECONFIG="`pwd`/config"
+export ARTIFACTS_DIR="`pwd`/artifacts"
+export CUSTOM_HOMEDIR=$ARTIFACT_DIR
+
+# This si one of the variables injected by ci-firewall. Its purpose is to allow scripts to handle uniqueness as needed
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
+
+# Add GOBIN which is the bin dir we created earlier to PATH so any binaries there are automatically available in PATH
+export PATH=$PATH:$GOBIN
+
+# Prep for integration/e2e
+shout "Building odo binaries"
+make bin
+
+# copy built odo to GOBIN
+cp -avrf ./odo $GOBIN/
+shout "getting ginkgo"
+make goget-ginkgo
+
+shout "Start minikube"
+# Delete minikube instance, if in anycase already exists
+minikube delete
+minikube start --vm-driver=docker --container-runtime=docker
+set +x
+# Get kubectl cluster info
+kubectl cluster-info
+
+set -x
+# Set kubernetes env var as true, to distinguish the platform inside the tests
+export KUBERNETES=true
+
+# Integration tests
+shout "Running integration Tests"
+make test-cmd-project
+make test-integration-devfile
```

Comment:
When do you clean up the openstack instances created with configure-openstack-vm.sh ? I had seen many instances left behind and eventually run out of space. I expect a call to delete the corresponding VM after tests are executed.

Reply:
> When do you clean up the openstack instances created with configure-openstack-vm.sh ?

Basically I am deleting the whole vm using `delete-openstack-vm.sh` on jenkins side itself once test run completed.

> I had seen many instances left behind and eventually run out of space.

This was the case when I was trying to implement minikube and fixing the failure. However it has been fixed just after a successful run of minikube on jenkins.